src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as os from 'os';
+import { IClipboardService } from 'sql/platform/clipboard/common/clipboardService';
+
+export function HandleCopyRequest(clipboardService: IClipboardService, range: Slick.Range, getCellValue: (row, cell) => string): void {
```

Comment:
`HandleCopyRequest` should be camelCased like `handleCopyRequest`

Reply:
good catch, i had the file name camel cased but forgot the method name...